client/notices_test.go

```diff
@@ -74,8 +80,10 @@ func (cs *clientSuite) TestNotices(c *C) {
 		"expire-after": "168h"
 	}, {
 		"id":   "2",
-  		"type": "warning",
+		"user-id": 0,
+		"type": "warning",
 		"key": "be careful!",
+		"visibility": "public",
```

Comment:
Shall we make this one "private" just to test that's being converted properly in the Notices code path?